relevanceai/dataset_api/dataset.py

```diff
@@ -137,6 +137,58 @@ def bulk_fn(documents):
             self.dataset_id, bulk_fn, select_fields=[self.field]
         )
 
+    def value_counts(
+        self,
+        normalize: bool = False,
+        ascending: bool = False,
+        sort: bool = False,
+        bins: Union[int, None] = None,
```

Comment:
```
bins: Optional[int] = None
```

Reply:
> not sure why mypy is failing?

It's failing because you told the user it's an integer but then you run non-integer operations like `count`. Do you want it to be an integer or a list?